src/Type/Php/StrRepeatFunctionReturnTypeExtension.php

```diff
@@ -42,6 +42,10 @@ public function getTypeFromFunctionCall(
 			return new ConstantStringType('');
 		}
 
+		if ($inputType instanceof ConstantStringType && $multiplierType instanceof ConstantIntegerType) {
+			return new ConstantStringType(str_repeat($inputType->getValue(), $multiplierType->getValue()));
```

Comment:
The code will crash when analysing code like this: https://3v4l.org/XRaOj

Reply:
good point. added a test and made it return `NeverType` on negative multiplier